modules/setting/config.go

```diff
@@ -49,6 +49,7 @@ func DefaultOpenWithEditorApps() OpenWithEditorAppsType {
 
 type RepositoryStruct struct {
 	OpenWithEditorApps *config.Value[OpenWithEditorAppsType]
+	SnippetRemoteName  *config.Value[string]
```

Comment:
Is `Snippet` a proper name?

IMO it is not a "snippet", it is for "git guide for users"